client/my-sites/stats/stats-notices/gdpr-cookie-consent-notice.tsx

```diff
@@ -0,0 +1,66 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import NoticeBanner from '@automattic/components/src/notice-banner';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import useNoticeVisibilityMutation from 'calypso/my-sites/stats/hooks/use-notice-visibility-mutation';
+import { StatsNoticeProps } from './types';
+
+const GDPRCookieConsentNotice = ( { siteId, isOdysseyStats }: StatsNoticeProps ) => {
+	const translate = useTranslate();
+	const [ noticeDismissed, setNoticeDismissed ] = useState( false );
+
+	const { mutateAsync: postponeNoticeAsync } = useNoticeVisibilityMutation(
+		siteId,
+		'gdpr_cookie_consent',
+		'postponed',
+		7 * 24 * 3600
+	);
+
+	const dismissNotice = () => {
+		isOdysseyStats
+			? recordTracksEvent( 'jetpack_odyssey_stats_gdpr_cookie_consent_notice_dismissed' )
+			: recordTracksEvent( 'calypso_stats_gdpr_cookie_consent_notice_dismissed' );
+
+		setNoticeDismissed( true );
+		postponeNoticeAsync();
+	};
+
+	if ( noticeDismissed ) {
+		return null;
+	}
+
+	const classNames = clsx(
+		'inner-notice-container has-odyssey-stats-bg-color',
+		! isOdysseyStats && 'inner-notice-container--calypso'
+	);
+
+	const bannerBody = isOdysseyStats
+		? translate(
+				'To fix, go to {{link}}Complianz settings{{/link}} use the toggle to disable Jetpack integration.',
+				{
+					components: {
+						link: <a href="/wp-admin/admin.php?page=complianz#integrations/integrations-plugins" />,
```

Comment:
![image](https://github.com/Automattic/wp-calypso/assets/1425433/f4423423-4125-4023-b084-7dc8d1704798)


Could we please use the admin_url to make the URL? Because there are cases where the admin URL is not standard, for example, placed in a subdirectory.

Reply:
Addressed in [a33158e](https://github.com/Automattic/wp-calypso/pull/91762/commits/a33158eaafa56a9b6375f262af67f5e5a244f2f7).